src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int ARRAY_SIZE = 6;
+    private static final FigureSupplier figureSupplier = new FigureSupplier();
```

Comment:
According to the checklist, you should not use static methods or fields for the `FigureSupplier` instance. Instead, create a new instance of `FigureSupplier` inside the `main` method.